tests/utils/cloudformation/index.js

```diff
@@ -0,0 +1,73 @@
+const AWS = require('aws-sdk');
+const _ = require('lodash');
+const region = require('../misc').testRegion;
+
+function findStacks(name, status) {
+  const CF = new AWS.CloudFormation({ region });
+
+  const params = {};
+  if (status) {
+    params.StackStatusFilter = status;
+  }
+
+  function recursiveFind(found, token) {
+    if (token) params.NextToken = token;
+    return CF.listStacks(params).promise().then(result => {
+      const matches = result.StackSummaries.filter(stack => stack.StackName.match(name));
+      if (matches.length) {
+        _.merge(found, matches);
```

Comment:
Wouldn't `found.push(...matches)` have same effect?

A side note: I have an impression that in ca 80% of places where we rely on lodash we can just use language features directly - and probably maintenance wise relying on native should be preferred

Reply:
Yes, good point. We had to do that because of Node.js 4.